### ISSUE TITLE
Compatibility with Alfresco Content Services 23.*

### ISSUE THREAD

**Issue by Karek-MA** [aldica/aldica#69]
I'm using Alfresco Content Services latest version installed manually (via zip files) on tomcat. I would like to use aldica to ensure high availability with 2 nodes. However, it looks like ALDICA  is not compatible with the latest version 23.* !?

**Comment by AFaust** [comment]
That is correct - I have been lacking the necessary time off of paid projects this year to work on a compatibility upgrade for this addon, though I have already worked on upgrading one of its dependencies. I hope to have time to work on this over the end/start of the year.
